WordPress/Classes/Services/MediaSettings.swift

```diff
@@ -3,14 +3,77 @@ import AVFoundation
 
 class MediaSettings: NSObject {
     // MARK: - Constants
+    fileprivate let imageOptimizationKey = "SavedImageOptimizationSetting"
     fileprivate let maxImageSizeKey = "SavedMaxImageSizeSetting"
+    fileprivate let imageQualityKey = "SavedImageQualitySetting"
     fileprivate let removeLocationKey = "SavedRemoveLocationSetting"
     fileprivate let maxVideoSizeKey = "SavedMaxVideoSizeSetting"
+    fileprivate let advertiseImageOptimizationKey = "SavedAdvertiseImageOptimization"
 
+    fileprivate let defaultImageOptimization = true
+    fileprivate let defaultMaxImageSize = 2000
+    fileprivate let defaultImageQuality: ImageQuality = .medium
+    fileprivate let defaultMaxVideoSize: VideoResolution = .sizeOriginal
+    fileprivate let defaultRemoveLocation = true
 
     fileprivate let minImageDimension = 150
     fileprivate let maxImageDimension = 3000
 
+    enum ImageQuality: String {
+        case maximum = "MaximumQuality100"
+        case veryHigh = "VeryHighQuality95"
+        case high = "HighQuality90"
+        case medium = "MediumQuality85"
+        case low = "LowQuality80"
+
+        var doubleValue: Double {
+            switch self {
+            case .maximum:
+                return 1.0
+            case .veryHigh:
+                return 0.95
+            case .high:
+                return 0.9
+            case .medium:
+                return 0.85
+            case .low:
+                return 0.8
+            }
+        }
+
+        var description: String {
+            switch self {
+            case .maximum:
+                return NSLocalizedString("Maximum", comment: "Indicates an image will use maximum quality when uploaded.")
+            case .veryHigh:
+                return NSLocalizedString("Very High", comment: "Indicates an image will use very high quality when uploaded.")
+            case .high:
+                return NSLocalizedString("High", comment: "Indicates an image will use high quality when uploaded.")
+            case .medium:
+                return NSLocalizedString("Medium", comment: "Indicates an image will use medium quality when uploaded.")
+            case(.low):
+                return NSLocalizedString("Low", comment: "Indicates an image will use low quality when uploaded.")
```

Comment:
80%, which is the current default, is [considered](https://sirv.com/help/articles/jpeg-quality-comparison/) optimal for web uploads. I would suggest going with 10% increments where 80% would be "Default", "Standard" or something like that.

Reply:
Thanks @kean for sharing this useful info 🙇 ! To be honest, I merely chose these quality values based on the ones we have in the Android version ([reference](https://github.com/wordpress-mobile/WordPress-Android/blob/41fd7a47cff9475ea1615da960c700b26a1033e6/WordPress/src/main/res/values/key_strings.xml#L366-L372)). However, as you suggested, it might be interesting to use different values and more close to standards:
- Very High: 100%.
- High: 90% (following [this article](https://sirv.com/help/articles/jpeg-quality-comparison/), this JPEG quality gives a very high-quality image while gaining a significant reduction on the original 100% file size).
- Medium (default): 80% (following [this article](https://sirv.com/help/articles/jpeg-quality-comparison/), this JPEG quality gives a greater file size reduction with almost no loss in quality).
- Low: 70% (following [this article](https://sirv.com/help/articles/jpeg-quality-comparison/), 75% JPEG quality and lower begins to show obvious differences in the image, which can reduce your website user experience).

Reply:
Suggestion applied in https://github.com/wordpress-mobile/WordPress-iOS/pull/21981/commits/b6f01271e0e5eda8b0c7d610bcbe270583883dfb.

Reply:
Note that I removed `Very High` option as the above four options should be enough to set different quality settings.